jobs/kpi-forecasting/kpi_forecasting/models/base_forecast.py

```diff
@@ -46,7 +45,7 @@ class BaseForecast:
 
     def __post_init__(self) -> None:
         # fetch observed observed data
-        self.collected_at = datetime.utcnow()
+        self.collected_at = datetime.now(timezone.utc)
```

Comment:
I think this changes `self.collected_at` to be timezone aware (e.g. `YYYY-MM-DD HH:MM:SS.000000+00:00`) instead of unaware (e.g. `YYYY-MM-DD HH:MM:SS.000000`). That seems reasonable, my only question is whether this impacts the write to BQ or the way the data will end up being stored.

Reply:
Ah good catch, I changed it only because `utcnow()` is deprecated so let me see if there is a unaware replacement

Reply:
Followed the directions here and just removed the tzinfo: https://blog.miguelgrinberg.com/post/it-s-time-for-a-change-datetime-utcnow-is-now-deprecated